### DISCUSSION TITLE
Total AV

### DISCUSSION THREAD

**Discussion by CaptainComeback** [DandelionSprout/adfilt#816] [Q&A] [answered]
I noticed Total AV is blocked I was just curious why it's blocked?

**Comment by DandelionSprout** [comment, marked as answer]
https://github.com/DandelionSprout/adfilt/blob/master/Dandelion%20Sprout's%20Anti-Malware%20List.txt#L1717-L1718 lists https://github.com/AdguardTeam/AdguardFilters/issues/91506#issuecomment-904080849 as the reasoning-comment for its blocking, and the reasoning was written in strong enough terms that I believe its writer (iam-py-test) on it.

**Reply by CaptainComeback** [reply]
Thanks for the reply. I will take your word for it. I can see that it says potential and that’s good enough for me. I was just curious and I probably would have installed it. So thanks again.

**Reply by iam-py-test** [reply]
I don't have sources offhand, but I often see this in spam/scam notifications, plus, it has a long history of shady behavior.

**Reply by CaptainComeback** [reply]
In case anyone else has the same question  I had I found another really good answer about how terrible Total AV is just watch this video.👇
https://youtu.be/PcS3EozgyhI

**Comment by CaptainComeback** [comment]
Thank you for the personal response I have heard the same thing from other sources and look what other antivirus companies have recently been caught doing. Have a good day.Sent from my iPhoneOn Mar 25, 2023, at 2:04 PM, iam-py-test ***@***.***> wrote:﻿
I don't have sources offhand, but I often see this in spam/scam notifications, plus, it has a long history of shady behavior.
